### DISCUSSION TITLE
Wall of Shame

### DISCUSSION THREAD

**Discussion by vhscom** [uBlockOrigin/uBlock-issues#1942] [General] [answered]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

Reddit is blocked for 300 million people in Indoneia and, ironically, isn't the most privacy-friendly website in the first place. As a result, I'm opening this feature request here on GitHub. Sorry if this doesn't fit your mold.

### A specific URL where the issue occurs

`https://css-tricks.com/lazy-loading-images-in-svelte/`

### Steps to Reproduce

Open Chris' site, and observe the number of trackers blocked.

### Expected behavior

I'd like to be able to see Chris' site along with others (ZeroHedge is probably a good example) that operate with gross disregard for user privacy listed in a Wall of Shame. The Wall of Shame ranks websites who are popular for whatever reason but are jam packed with trackers into a list uBlock users can talk about and openly mock.

### Actual behavior

There is no Wall of Shame I'm aware of. But this seems like a nice place for discussions about privacy for the hundreds of millions of users who cannot or don't want to use Reddit.

### uBlock Origin version

latest

### Browser name and version

LibreWolf

### Operating System and version

Arch Linux 5.15.11-3-t2

**Comment by ghost** [comment]
Why not here: https://github.com/uBlockOrigin/uBlock-issues/discussions ?

**Reply by vhscom** [reply]
I didn't see discussions available when I dropped in here originally. Looks like they're here now. :face_in_clouds:

**Comment by uBlock-user** [comment]
Converted to Discussion.

**Comment by vhscom** [comment]
Okay. Here's a first shot at the design. Create script that scrapes logs at the user's preferred interval and creates a summary report sorted by blocked count descending.

Summary includes:

- TLD including scheme
- Number pages visited on that TLD
- Aggregate block count across all pages

Summary does not include:

- Sites with less than 2 unique URLs visited
- The bottom 80% of the report (which is truncated)

Sites appearing on that list are then weighted using Alexa scores:
https://www.alexa.com/siteinfo/css-tricks.com

Sites with a high block count and a high Alexa score may be promoted to the Wall of Shame at the users descretion at specified time interval using a color change in the notification count in the extension.

**Comment by gorhill** [comment, marked as answer]
> scrapes logs

uBO does not keep logs.

**Comment by iam-py-test** [comment]
This would be hard, and kind-of a waste of the uBo team's time. Feel free to make your own 
It would take forever manually, and it's hard to measure using automation.
Just a note, not all the requests blocked by uBo _are_ trackers
